src/OctoshiftCLI.Tests/Octoshift/Services/GithubApiTests.cs

```diff
@@ -3332,4 +3332,58 @@ public async Task GetEnterpriseServerVersion_Returns_Version()
             .Replace("\\n", "")
             .Replace("\\t", "")
             .Replace(" ", "");
+
+    [Fact]
+    public async Task AbortMigration_Returns_Boolean()
```

Comment:
nit: Could you move this test method before the `Compact` method. IMO it's aesthetically more pleasant to have all the `private` methods after all `public` test methods.

Reply:
[e226bdd](https://github.com/github/gh-gei/pull/1162/commits/e226bddab3e23c901508ee9b0df81e6f73642196)